libfreerdp/freerdp.pc.in

```diff
@@ -3,6 +3,11 @@ exec_prefix=@CMAKE_INSTALL_PREFIX@
 libdir=@CMAKE_INSTALL_FULL_LIBDIR@
```

Comment:
`libdir` could also use `${prefix}`  or do I miss something here?

Reply:
No, quoting here https://cmake.org/cmake/help/v3.0/module/GNUInstallDirs.html:

```
Each CMAKE_INSTALL_FULL_<dir> value contains an absolute path constructed from the corresponding destination by prepending (if necessary) the value of CMAKE_INSTALL_PREFIX.
```

And that also seems to work here (have the prefix `/usr` here): `pkg-config freerdp3 --variable libdir` results here into `/usr/lib`, which is correct.

Reply:
@pnowack wrong doc, this is a `pkg-config` file that is generated here.

Reply:
@akallabeth didn't change that in the PR but you are correct that it could use prefix as well. I don't think it would make any difference because the path set for libdir is the path compiled in as search path but I'll change this